### DISCUSSION TITLE
Wide screen support

### DISCUSSION THREAD

**Discussion by RichiH** [thuliteio/doks#214] [Q&A]
I am currently evaluating Hugo themes. Doks is on my shortlist, but I will only choose a theme that has wide-screen support (and supports non-JS browsing through w3m & screen readers, which Doks does an OK job at already).

While this is an extreme example, it shows the problem nicely at 5120x1440:
![image](https://user-images.githubusercontent.com/754723/113690854-26009800-96cc-11eb-9c19-25beed1a7189.png)

**Comment by h-enk** [comment]
Hi @RichiH, thanks for checking out Doks!

Doks uses Bootstrap 5, so you can easily customize [breakpoints](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints) + [container widths](https://getbootstrap.com/docs/5.0/layout/containers/#sass) in `./assets/scss/common/_variables.scss`.

Breakpoint defaults:

```scss
$grid-breakpoints: (
  xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px
);
```

Container width defaults:

```scss
$container-max-widths: (
  sm: 540px,
  md: 720px,
  lg: 960px,
  xl: 1140px,
  xxl: 1320px
);
```

Hope this helps!

**Comment by RichiH** [comment]
Nice, thanks!
